Dig force aware procedures

AstroStucky opened the pull request:
## Sibling PR
[ow_simulator #364](https://github.com/nasa/ow_simulator/pull/364)

## Purpose
With the Balovnev model producing a realistic dig force we can no longer dig as deep as we like. Additionally, the new dig force trajectories may have broken some Plexil plans. This branch ensures all current Plexil plans still work with the new dig model epic branch in ow_simulator. 

## Changes
* Updated the following plans to work well with new dig force model:
  * TestActions.plp 
  * Demo.plp
  * ReferenceMission1.plp
  * ReferenceMission2.plp

## Testing
Run each of the modified plans, and ensure they work. Optionally you can run each of the plans with ow_simulator on the branch from [ow_simulator #365](https://github.com/nasa/ow_simulator/pull/365)

kmdalal commented:
I found the cause of the GuardedMove failures -- and many other insidious failures.  The fix is easy and will be done tomorrow.

kmdalal reviewed (approved):
Was this meant to still be on hold?  If so, re-request my review when it's ready.
The plans work great!  I fixed EuropaMission and added it to this branch.

AstroStucky commented:
> Was this meant to still be on hold? If so, re-request my review when it's ready. The plans work great! I fixed EuropaMission and added it to this branch.

Thanks! I think I glanced through that plan, but I thought it currently was not runnable, so I ignored it.
